### PR TITLE
#6033 - page editor button field label updates

### LINKED ISSUE #6033: Slice 2: Label updates
### Summary of change

- Update labels for certain inputs
    - Variant → Button Style
    - Title → Button Label
    - Layout/Style → Label Style

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- Closes #6033 

## Demo
image

## Checklist

- [ ] Designate a primary reviewer @turbochef

### CODE REVIEW

File: src/components/documentBuilder/edit/getElementEditSchemas.tsx [hunk 1]
```diff
@@ -21,11 +21,14 @@ import { type DocumentElementType } from "@/components/documentBuilder/documentB
 import React from "react";
 import { VALID_HEADER_TAGS } from "@/components/documentBuilder/allowedElementTypes";
 
-function getClassNameEdit(elementName: string): SchemaFieldProps {
+function getClassNameEdit(
+  elementName: string,
+  { label = "Layout/Style" }: { label?: string } = {}
```
Comment (turbochef) [inline src/components/documentBuilder/edit/getElementEditSchemas.tsx:26]: Why all of this instead of just:
```suggestion
  label: string
```

Comment (grahamlangford) [reply src/components/documentBuilder/edit/getElementEditSchemas.tsx:26]: Future-proofing. I tend to prefer options object args over single variables.

Comment (turbochef) [reply src/components/documentBuilder/edit/getElementEditSchemas.tsx:26]: In that case could we turn all the params into a single object? Easier for typing and easier to read.

**Review by turbochef — APPROVED** [review]
Non blocking opinion 👍

**Pull request merged**